Apps/Contoso.Forms.Puppet/Contoso.Forms.Puppet/App.xaml.cs

```diff
@@ -46,22 +46,6 @@ public App()
             RealUserMeasurements.SetRumKey("b1919553367d44d8b0ae72594c74e0ff");
             AppCenter.Start($"uwp={UwpKey};android={AndroidKey};ios={IosKey}",
                                typeof(Analytics), typeof(Crashes), typeof(Distribute), typeof(Push), typeof(RealUserMeasurements));
-
```

Comment:
actually we will need to test all of that again not to break compat, the analytics enabled method will be restored

Reply:
we better either keep google-services.json and set analytics in forms puppet, or move that to the non forms puppet